package.json

```diff
@@ -21,32 +21,32 @@
     "validate"
   ],
   "dependencies": {
-    "@babel/cli": "^7.13.16",
-    "@babel/core": "^7.14.0",
-    "@babel/preset-env": "^7.14.1",
-    "@babel/preset-react": "^7.13.13",
+    "@babel/cli": "^7.17.6",
+    "@babel/core": "^7.17.5",
+    "@babel/preset-env": "^7.16.11",
+    "@babel/preset-react": "^7.16.7",
```

Comment:
@douglasawh You must have run `npm install` or `npm i` instead of `npm ci`. This time around, it's fine, but in the future use `npm ci`. It uses the lock file to install the npm packages rather than doing an install and upgrading packages, if there are minor or patch updates available.

Reply:
when you run the npm ci, it tells you to update your packages, or more precisely, it says this:

```
33 vulnerabilities (24 moderate, 9 high)

To address issues that do not require attention, run:
  npm audit fix

To address all issues (including breaking changes), run:
  npm audit fix --force

Run `npm audit` for details.
```

I doubt anyone else is going to be working on this, but it is probably worth updating the README, which I will get to at some point.

Reply:
Ok, yeah, but that doesn't mean you automatically update everything based on that. Especially in enterprise, the "fixes" need to be researched a bit.